ezbake/config/logback.xml

```diff
@@ -18,6 +18,7 @@
 
     <root level="info">
         <!--<appender-ref ref="STDOUT"/>-->
+        <appender-ref ref="${logappender:-DUMMY}"/>
```

Comment:
Ha yep. I guess this sort of pattern was copied from PuppetDB's subcommands. I actually think that we might be able to move this default value into the foreground command itself rather than requiring every logback.xml file to specify it this way.

I'm going to try this out locally, and will update with my findings.